packages/lib/src/link/connectors.ts

```diff
@@ -9,6 +9,25 @@ import { logger } from "../logging";
 import ExponentialBackoff from "../ExponentialBackoff";
 import type { Request, Event } from "./link";
 
+/**
+ * Numbered codes describing why a connection was closed
+ * See: https://github.com/Luka967/websocket-close-codes
+ */
+export enum ConnectionClosed {
+	// Codes between 1000 and 1015 are pre-allocated for web sockets
+	Normal = 1000, // CLOSE_NORMAL
+	ServerQuit = 1001, // CLOSE_GOING_AWAY
+	ProtocolError = 1002, // CLOSE_PROTOCOL_ERROR
+	PolicyError = 1008, // Policy Violation
+	ServerError = 1011, // Server Error
```

Comment:
Shouldn't this use the standard namings that are available in the registry? see https://www.iana.org/assignments/websocket/websocket.xhtml#close-code-number

Normal -> NormalClosure
ServerQuit -> GoingAway
PolicyError -> PolicyViolation
ServerError -> InternalError

Reply:
Orginaly I picked names based on the error messages we already used. But I'm not attached to them so I'll update to iana standard.